samples/CommunityToolkit.Maui.Sample/ViewModels/Views/CameraView/CameraViewModel.cs

```diff
@@ -0,0 +1,96 @@
+﻿using System.ComponentModel;
+using CommunityToolkit.Maui.Core;
+using CommunityToolkit.Maui.Core.Primitives;
+using CommunityToolkit.Maui.Views;
+using CommunityToolkit.Mvvm.ComponentModel;
+using CommunityToolkit.Mvvm.Input;
+
+namespace CommunityToolkit.Maui.Sample.ViewModels.Views;
+
+public partial class CameraViewModel() : BaseViewModel
+{
+	[ObservableProperty]
+	CameraFlashMode flashMode;
+
+	[ObservableProperty]
+	CameraInfo? selectedCamera;
+
+	[ObservableProperty]
+	Size selectedResolution;
+
+	[ObservableProperty]
+	float currentZoom;
+
+	[ObservableProperty]
+	string cameraNameText = "", zoomRangeText = "", currentZoomText = "", flashModeText = "", resolutionText = "";
+
+	public ICollection<CameraFlashMode> FlashModes { get; } = Enum.GetValues<CameraFlashMode>();
+
+	partial void OnFlashModeChanged(CameraFlashMode value)
+	{
+		UpdateFlashModeText();
+	}
+
+	partial void OnCurrentZoomChanged(float value)
+	{
+		UpdateCurrentZoomText();
+	}
+
+	partial void OnSelectedResolutionChanged(Size value)
+	{
+		UpdateResolutionText();
+	}
+
+	[RelayCommand]
+	async Task CaptureImage(CameraView cameraView, CancellationToken token)
+	{
+		await cameraView.CaptureImage(token);
```

Comment:
This currently breaks MVVM, is this fine for a v1 or do we want to consider exposing `Command`s on the `CameraView` to support the MVVM architecture more cleanly?

Reply:
> breaks MVVM

Because we calling the View in the ViewModel? Agreed. 

Would the solution be to add a `public ICommand CaptureImageCommand { get; set; }`?

Reply:
Yes along those lines but it wants to be readonly. I will add those now

Reply:
I've added them in now. @brminnick would you be willing to review that change?